tools/debug-ui/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:19-alpine AS build
```

Comment:
Not everyone might be familiar with Docker and/or nginx. 
Could you add code script instructions into the README.md on how to use this and run this locally?
Thanks!